docs/migration-guides/prometheus-migration.md

```diff
@@ -0,0 +1,340 @@
+
+![prometheus_successful_migration](../../images/prometheus-migration/tsdb_restored.png)
+
+## Step 4: Create a TSDB snapshot and restore it in the new deployment
+
+Now that the process has been explained in detail, these are the commands that will have to be executed in order to migrate the TSDB from the old Prometheus deployment to the BKPR one.
+
+After a previous sync with the TSDB, the command to generate the Prometheus snapshot is:
+
+`curl -XPOST http://localhost:9090/api/v1/admin/tsdb/snapshot?skip_head=true`
+
+That command is meant to be executed from the sidecar rsync container that is deployed in the old Prometheus deployment. As soon as you execute that command, the snapshot will be stored under the data directory of Prometheus, under `snapshots/`.
+
+Once the snapshot is done, connect to the sidecar rsync container in the new Prometheus deployment and execute:
+
+`rsync -avrP prometheus-rsync.NAMESPACE.svc.cluster.local::data/snapshots/ /opt/bitnami/prometheus/data/`
+
+> Note:  Make sure you change the NAMESPACE placeholder for the Kubernetes namespace where the old Prometheus deployment lives.
+
+After the rsync has finished, move the block directories to the main data directory of Prometheus. This an example of the structure you will find data directory of the new Prometheus deployment:
+
+```bash
+/data # ls -la
+total 96
+drwxrwsr-x   24 1001     1001          4096 Dec  5 11:00 .
+drwxr-xr-x    1 root     root          4096 Nov 30 13:02 ..
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CTTC16QP4NCPRM28H7Y6HXWB
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CV05DT3E6QKSEMR2TAEEYWTQ
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CV5YTF6Z4F22DJ5ECNM4PAH9
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CVBR71YE4M97K3SDXXQ9449W
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CVHHKMAEH0A0WP7BSYCKDTST
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CVQB0GF1153DSTGSQ8HJ82KF
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CVX4CTZJ8VDVS2PQKQQYZVRN
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CW2XSE758NYFKWY2AX3ZGRMY
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CW8Q6AZPZKD10ZFFNP102V4Z
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CWEGJQ52W2YJXZ86J51FJGBP
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CWM9ZAP7SBZBXQJPT4409PE6
+drwxrwsr-x    3 1001     1001          4096 Nov 30 13:00 01CWT3C1RRY2SEZT93AX0EFYW0
+-rw-rw-r--    1 1001     1001             0 Nov 30 12:05 lock
+drwxrwsr-x    3 1001     1001          4096 Dec  5 12:01 wal
+```
+
+Now, delete the Prometheus pod to force a reload of all the TSDB blocks with:
+
+`kubectl delete pod prometheus-0 -n kubeprod --grace-period=0`
+
+In the next start
```

Comment:
On the next restart, you should see something like this in the Prometheus logs:

Reply:
done